01-node-tutorial/answers/09-path-module.js

```diff
@@ -0,0 +1,11 @@
+const path = require('path');
+
+const file = path.join('node-express-course', '01-node-tutorial','answers','09-path-module.js');
+
+//console.log(file);
```

Comment:
Super minor but it folks will generally call out commented out console logs in PRs to be removed. It is common to need to log something while developing but if it is not needed long-term it is generally good practice to remove it.